lib/htmlValidator.js

```diff
@@ -53,9 +52,24 @@ module.exports = function(app, callback) {
           }
         });
 
+        validatorProcess.stderr.on('data', (data) => {
+          if (`${data}`.includes('No Java runtime present, requesting install.')) {
+            console.error('There was an error in authentication. Please make sure to have Java installed.');
+            reject();
+          }
+          else {
+            resolve();
+          }
+        });
+
         validatorProcess.on('error', (err) => {
-          console.error(`error: ${err}`);
-          reject();
+          if (err.code === 'ENOENT' ) {
+            console.error('error: You must install Java to continue. Warning: HTML validation disabled, error on initialization (check to make sure Java is installed and in your path)');
```

Comment:
this seems kinda redundant. there's a warning and an error saying the same thing twice on one line... pick one?